tests/notebooks/cpu/training/requirements.in

```diff
@@ -1,3 +1,3 @@
-# Pinning to 1.6.0 because of the breaking change with the imports in 1.7.0 https://github.com/canonical/charmed-kubeflow-uats/issues/84
+# pinning to 1.6.0 because of import problems for higher versions https://github.com/canonical/charmed-kubeflow-uats/issues/164
```

Comment:
Let's either close #84 or link both issues here, I think it'd be worth it

Reply:
added both